src/esp/gfx/Renderer.cpp

```diff
@@ -63,9 +63,33 @@ struct Renderer::Impl {
           DepthShader::Flag::UnprojectExistingDepth);
     }
 
+    RenderTarget::Flags renderTargetFlags_ = {};
+    switch (sensor.specification()->sensorType) {
+      case sensor::SensorType::Color:
```

Comment:
Can you add an assert here that `NoTextures` isn't set.  Right now a user could make a RGBA RenderTarget when there aren't textures.

Reply:
@erikwijmans : Just updated the code to address your concern here. Take a look.

Reply:
Though ... what if the intent is to render just vertex colors? That can be thought of as some kind of an optimization as well, ignoring textures and using vertex colors instead. Or is that not the intent of this functionality?

Reply:
Currently the intent of this is to not load textures when the agent doesn't have and rgb sensor.  AFAIK, we don't have any assets that have both textures and vertex coloring so I think tying theses makes sense, but this is a good idea that we should keep in mind for the future.

Reply:
In computer graphics community, we played with the meshes without textures quite often, e.g., the Stanford bunny. 

But I guess it would not be the case here. :)

Reply:
Agreed that is a legitimate use for rgb rendering without textures in general :)